kategory-docs/docs/docs/datatypes/try/README.md

```diff
@@ -4,6 +4,182 @@ title: Try
 permalink: /docs/datatypes/try/
 ---
 
-## Try 
+## Try
+
+Kategory has [lots of different types of error handling and reporting](http://kategory.io/docs/patterns/error_handling/), which allows you to choose the best strategy for your situation. 
+
+For example, we have `Option` to model the absence of a value, or `Either` to model the return of a function as a type that may have been successful, or may have failed. 
+
+On the other hand, we have `Try`, which represents a computation that can result in an `A` result (as long as the computation is successful) or in an exception if something has gone wrong. 
+
+That is, there are only two possible implementations of `Try`: a `Try` instance where the operation has been successful, which is represented as `Success<A>`; or a `Try` instance where the computation has failed with a `Throwable`, which is represented as `Failure<A>`.
+
+With just this explanation you might think that we are talking about an `Either<Throwable, A>`, and you are not wrong. `Try` can be implemented in terms of `Either`, but its use cases are very different.
+
+If we know that an operation could result in a failure, for example, because it is code from a library over which we have no control, or better yet, some method from the language itself. We can use `Try` as a substitute for the well-known `try-catch`, allowing us to rise to all its goodness.
+
+The following example represents the typical case when consuming Java code, where domain errors are represented with exceptions.  
+
+```kotlin:ank
```

Comment:
Add silent to this block to avoid getting a blank return

Reply:
https://github.com/kategory/kategory/pull/458/commits/2d7c47de71d5ea808386b2f3e7ccfa2b08fd1193